internal/pkg/service/buffer/storage/local/writer/writer_test.go

```diff
@@ -0,0 +1,714 @@
+package writer
+
+import (
+	"context"
+	"github.com/c2h5oh/datasize"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/compression"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/local"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/local/writer/allocate"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/local/writer/disksync"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/local/writer/test"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/staging"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model/column"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/utctime"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+	"github.com/keboola/keboola-as-code/internal/pkg/validator"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+)
+
+func TestVolume_NewWriterFor_Ok(t *testing.T) {
+	t.Parallel()
```

Comment:
Ostatne testy by snad mali byt jasne.

Trocha to je duplicitne s testami na nizsej urovni, 
potrebujeme hlavne overi, ze sa nastavenia zo `*storage.Slice` spravne prejavia  v realite.